tests/test_config.py

```diff
@@ -8,9 +8,16 @@
 from pluggy import PluginManager
 
 import tox
-from tox.config import CommandParser, DepOption, SectionReader, get_homedir
-from tox.config import get_version_info, getcontextname, is_section_substitution, parseconfig
-
+from tox.config import (
+    CommandParser,
+    DepOption,
+    SectionReader,
+    get_homedir,
+    get_version_info,
+    getcontextname,
+    is_section_substitution,
+    parseconfig,
+)
```

Comment:
argh, now we're back to that thing, but tbh atm I don't care anymore. From now on it's just a question of reconfiguring whatever formatter we use.

Reply:
yeah, maybe things will change in future with https://github.com/ambv/black/issues/204

Reply:
stays as it is then :+1: